### ISSUE TITLE
[Bug]: In high contrast mode, focus indicator is not visible on the column headers of the table in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#10415]
### Link to reproducible scenario

https://www.ag-grid.com/examples/themes/built-in-themes/reactFunctional/

### Describe the bug

Steps to reproduce 

1. Open URL: [React Example - Themes - Built in Themes] (https://www.ag-grid.com/examples/themes/built-in-themes/reactFunctional/) on latest Edge browser. 
2. Table will appear.
3. Press tab key to reach to the 'Make' column header of table.
4. Verify whether in high contrast mode keyboard focus is visible on the column headers of the table or not.

Actual behaviour
On pressing tab, in high contrast mode focus indicator is not visible on the column headers of the table in Windows.

https://github.com/user-attachments/assets/9bb6e0bf-7304-4281-9827-7d5d4994da53

<img width="719" alt="Image" src="https://github.com/user-attachments/assets/74c8cd2d-51e3-45c3-a64b-6fb0f227de23" />
<img width="625" alt="Image" src="https://github.com/user-attachments/assets/06e10616-9d2b-4b8f-b895-b793f3fe73c6" />
<img width="959" alt="Image" src="https://github.com/user-attachments/assets/fb73c124-33d4-4d0d-b98d-b67432f3904f" />

Expected behaviour
On pressing tab, in high contrast mode focus indicator should be visible on the column headers of the table in Windows.

More information
Title: In high contrast mode, focus indicator is not visible on the column headers of the table in Windows.
User Impact: Low vision users are not able to identify the currently focused control if focus indicator is not visible on receiving focus.

### Version

33.1.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by ava-utting** [comment]
Hi @msftedad ,

Thank you for raising this with us.

Currently there's no way to implement this behavior with AG Grid. However this is a known area which we are looking to improve and we have a feature request logged to address it. This item is being tracked with the following reference: AG-6163.

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind Regards,
Ava
